tests/e2e/manifest/target-virtualService.yaml

```
@@ -0,0 +1,19 @@
+apiVersion: networking.istio.io/v1beta1
+kind: VirtualService
+metadata:
+  name: target-deployment
+  namespace: istio-system
+spec:
+  hosts:
+  - "*" 
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)